pkg/cmd/server/kubernetes/master.go

```diff
@@ -83,6 +82,12 @@ func (c *MasterConfig) InstallAPI(container *restful.Container) []string {
 	}
 }
 
+func (c *MasterConfig) RunNamespaceController() {
+	namespaceController := namespace.NewNamespaceManager(c.KubeClient)
+	namespaceController.Run(1 * time.Minute)
```

Comment:
This means you will see a namespace appear in "Terminating" status for ~1 min.  The value is the same default as upstream.  You are unable to create content in a terminating namespace.  There are two controllers that need to run in order to delete content (the k8s one, and the openshift one) so it will take ~1-2 minutes for something to truly be gone.